src/sciline/visualize.py

```diff
@@ -21,6 +21,14 @@
 from .typing import Graph, Item, Key, get_optional
 
 
+class GraphVisualization(Digraph):  # type: ignore[misc]
+    def _repr_mimebundle_(self, *args, **kwargs):  # type: ignore[no-untyped-def]
+        data = super()._repr_mimebundle_(*args, **kwargs)
+        if 'image/svg+xml' in data:
+            data['text/html'] = data['image/svg+xml']
```

Comment:
Can you add a comment explaining why we are doing this? I think it is not immediately obvious for future readers.